### PR TITLE
Stage release 1.5.4

### PR DESCRIPTION
**Pull request by jmurret**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+## 1.5.4 (November 1, 2024)
+
+SECURITY:
+
+* Upgrade Go to use 1.22.7. This addresses CVE
+  [CVE-2024-34155](https://nvd.nist.gov/vuln/detail/CVE-2024-34155) [[GH-608](https://github.com/hashicorp/consul-dataplane/pull/608)]
+* Upgrade to support Envoy `1.29.10`. [[GH-664](https://github.com/hashicorp/consul-dataplane/pull/664)]
```
[the remark this answers is not in the document]
Comment (nathancoleman) [reply CHANGELOG.md:7]: Should be https://github.com/hashicorp/consul-dataplane/pull/660

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+
+## 1.5.3 (August 30, 2024)
+
+IMPROVEMENTS:
+
+* Update `Update `github.com/hashicorp/consul/proto-public` to v0.6.2. [[GH-585](https://github.com/hashicorp/consul-dataplane/pull/585)]
```
Comment (nathancoleman) [inline CHANGELOG.md:13]:
```suggestion
* Update `github.com/hashicorp/consul/proto-public` to v0.6.2. [[GH-585](https://github.com/hashicorp/consul-dataplane/pull/585)]
```

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmurret) [reply CHANGELOG.md]: thank you.  fixed in 58396932684eb7a2e67df5f3a690c44cec36ce65

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+
+* Update `github.com/hashicorp/consul/proto-public` to v0.6.2. [[GH-585](https://github.com/hashicorp/consul-dataplane/pull/585)]
+* Update multiple dependencies. [[GH-602](https://github.com/hashicorp/consul-dataplane/pull/602)]
+* Update ubi base image to `ubi9-minimal:9.4`. [[GH-602](https://github.com/hashicorp/consul-dataplane/pull/602)]
+
+## 1.5.2 (August 28, 2024)
+
+IMPROVEMENTS:
+
+* Update `github.com/hashicorp/consul/proto-public` to v0.6.2. [[GH-585](https://github.com/hashicorp/consul-dataplane/pull/585)]
```
Comment (nathancoleman) [inline CHANGELOG.md:21]: Should this entry be in both 1.5.2 and 1.5.3?

Comment (jmurret) [reply CHANGELOG.md:21]: let me look.  i copied this from our GH releases because they were missing.

Comment (jmurret) [reply CHANGELOG.md]: fixed this. this release was redacted

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+* Update multiple dependencies. [[GH-602](https://github.com/hashicorp/consul-dataplane/pull/602)]
+* Update ubi base image to `ubi9-minimal:9.4`. [[GH-602](https://github.com/hashicorp/consul-dataplane/pull/602)]
+
+## 1.5.2 (August 28, 2024)
+
+Release redacted, use `1.5.3`
+
+
```
Comment (jmurret) [inline CHANGELOG.md:21]:
```suggestion
```

**Review by nathancoleman — APPROVED** [review]

**Pull request merged**